tests/unit/amo/components/TestCollection.js

```diff
@@ -1094,51 +1144,7 @@ describe(__filename, () => {
     const root = renderComponent({
       editing: true,
       errorHandler,
-      location: fakeRouterLocation({ query: { page } }),
-      store,
-    });
-
-    fakeDispatch.resetHistory();
-
-    // This simulates the user clicking the "Remove" button on the
-    // EditableCollectionAddon component.
-    root.instance().removeAddon(addonId);
-    sinon.assert.callCount(fakeDispatch, 1);
-    sinon.assert.calledWith(
-      fakeDispatch,
-      removeAddonFromCollection({
-        addonId,
-        errorHandlerId: errorHandler.id,
-        page,
-        slug: collectionDetail.slug,
-        username: collectionDetail.author.username,
-      }),
-    );
-  });
-
-  it('dispatches removeAddonFromCollection when removeAddon is called without a page defined', () => {
```

Comment:
This test, as well as all the others that are for `dispatches xxx when yyy is called without a page defined` was removed because there is no longer a condition where the component doesn't have a `page` prop. It will always have a `filters` prop and that it always passed, so there's no need for 2 tests for each of these cases (one with a `page` prop and one without).